addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -133,16 +133,43 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 .o_wprofile_all_users_nav {
     border-width: 1px 0;
 
-    &, .o_wprofile_course_nav_search {
+    &, .o_wprofile_course_nav_search, .o_wprofile_all_users_nav_btn {
         background-color: rgba(white, 0.05);
         border-color: rgba(white, 0.1);
         border-style: solid;
     }
 
-    .o_wprofile_course_nav_search {
+    .o_wprofile_course_nav_search, .o_wprofile_all_users_nav_btn {
         border-width: 0 1px;
     }
 
+    .o_wprofile_all_users_nav_btn {
+        @include media-breakpoint-up(md) {
+            @include o-hover-text-color(white, $gray-800);
+            margin-top: -1px;
+            border-radius: 0;
+            min-height: 35px;
+
+            &:hover {
+                background-color: white;
+            }
+        }
+    }
+
+    .o_wprofile_all_users_nav_btn_container {
+        @include media-breakpoint-down(sm) {
+            ~ .d-md-none {
+                padding-left: 0;
+            }
+
+            .o_wprofile_all_users_nav_btn {
+                @include o-hover-text-color(white, white);
```

Comment:
I got your point as a general approach, but in this case I'm not picking a color because of it's position in the grayscale (eg. use a color lighten than the 'gray-500' background)... it's an arbitrary choose.
Even if we will decide that 'white' is not '#000", the right color for this element will remain white anyway

Reply:
I don't see why but ok